### PR TITLE
i18n(ru): Update `why-astro.mdx` translation

### PR DESCRIPTION
**Pull request by mearashadowfax**
#### Description

This PR updates the Russian translation of  `why-astro.mdx`  page, addressing outdated content flagged on the Translation Tracker.

### CODE REVIEW

File: src/content/docs/ru/concepts/why-astro.mdx [hunk 1]
```diff
@@ -1,135 +1,90 @@
...
-Напротив же, большинство веб фреймворков предназначены для создания веб-приложений.
-Эти фреймворки лучше всего подходят для создания более сложных приложений в браузере: панелей администратора,
-почтовых сервисов, социальных сетей, списков задач и даже нативных приложений, таких как [Figma](https://figma.com/) 
-и [Ping](https://ping.gg/).
+Вот пять основных принципов дизайна, которые помогут объяснить, почему мы создали Astro, какие проблемы он решает и почему Astro может быть лучшим выбором для вашего проекта или команды.
 
-Это одно из самых важных различий, которые необходимо понять в Astro.
-Уникальный фокус на контент позволяет Astro находить компромиссы и предоставлять непревзойденные функции 
-производительности, которые не имели бы смысла внедрять во веб-фреймворках более ориентированных на веб-приложения.
+Astro...
```
Comment (baileys-li) [inline src/content/docs/ru/concepts/why-astro.mdx:26]: There is a typography symbol `…` on current `en` page. By the typography rules of Russian lang the same symbol is required too
```suggestion
Astro…
```

File: src/content/docs/ru/concepts/why-astro.mdx [hunk 2]
```diff
@@ -1,135 +1,90 @@
...
 title: Почему Astro?
-description: >-
-  Astro это веб-фреймворк все-в-одном, для разработки быстрых, ориентированных
-  на контент вебсайтов. Узнать больше.
+description: "Astro - это веб-фреймворк для создания веб-сайтов, ориентированных на контент, таких как блоги, маркетинг и электронная коммерция. Узнайте, почему Astro может стать хорошим выбором для вашего следующего веб-сайта."
 i18nReady: true
 ---
 
-Astro это **веб-фреймворк все-в-одном**, для разработки **быстрых, ориентированных на контент вебсайтов.**
+**Astro** - это веб-фреймворк для создания **веб-сайтов, ориентированных на контент**, таких как блоги, маркетинг и электронная коммерция. Astro наиболее известен своим новаторским [фронтенд-архитектурным подходом](/ru/concepts/islands/), направленным на снижение нагрузки и сложности JavaScript по сравнению с другими фреймворками. Если вам нужен сайт, который быстро загружается и имеет отличные показатели SEO, то Astro - это то, что вам нужно.
```
Comment (baileys-li) [inline src/content/docs/ru/concepts/why-astro.mdx:7]: [Тире is required](https://orfogrammka.ru/%D1%82%D0%B8%D0%BF%D0%BE%D0%B3%D1%80%D0%B0%D1%84%D0%B8%D0%BA%D0%B0/%D1%80%D0%B0%D0%B7%D0%BB%D0%B8%D1%87%D0%B8%D0%B5_%D1%82%D0%B8%D1%80%D0%B5_%D0%B8_%D0%B4%D0%B5%D1%84%D0%B8%D1%81%D0%B0/)
```suggestion
**Astro** — это веб-фреймворк для создания **веб-сайтов, ориентированных на контент**, таких как блоги, маркетинг и электронная коммерция. Astro наиболее известен своим новаторским [фронтенд-архитектурным подходом](/ru/concepts/islands/), направленным на снижение нагрузки и сложности JavaScript по сравнению с другими фреймворками. Если вам нужен сайт, который быстро загружается и имеет отличные показатели SEO, то Astro - это то, что вам нужно.
```

Not only in this line

File: src/content/docs/ru/concepts/why-astro.mdx [hunk 3]
```diff
@@ -1,135 +1,90 @@
...
+
+Хорошая производительность важна всегда, но это *особенно* важно для веб-сайтов, ориентированных на контент. Хорошо доказано, что низкая производительность приводит к потере вовлеченности, конверсии и денег. К примеру:
 
 - Каждые сэкономленные 100мс → на 1% больше конверсии ([Mobify](https://web.dev/why-speed-matters/))
 - На 50% быстрее → на 12% больше продаж ([AutoAnything](https://www.digitalcommerce360.com/2010/08/19/web-accelerator-revs-conversion-and-sales-autoanything/))
 - На 20% быстрее → на 10% больше конверсии ([Furniture Village](https://www.thinkwithgoogle.com/intl/en-gb/marketing-strategies/app-and-mobile/furniture-village-and-greenlight-slash-page-load-times-boosting-user-experience/))
 - На 40% быстрее → на 15% больше регистраций ([Pinterest](https://medium.com/pinterest-engineering/driving-user-growth-with-performance-improvements-cfc50dafadd7))
-- 850ms → на 7% больше конверсии ([COOK](https://web.dev/why-speed-matters/))
-- Каждая 1 секунда → на 10% меньше пользователей ([BBC](https://www.creativebloq.com/features/how-the-bbc-builds-websites-that-scale))
+- На 850ms быстрее → на 7% больше конверсии ([COOK](https://web.dev/why-speed-matters/))
```
Comment (baileys-li) [inline src/content/docs/ru/concepts/why-astro.mdx:60]: Why not translate units? Not sure about COOK meaning
```suggestion
- На 850 мс быстрее → на 7% больше конверсии ([Рецепт](https://web.dev/why-speed-matters/))
```

File: src/content/docs/ru/concepts/why-astro.mdx [hunk 4]
```diff
@@ -1,135 +1,90 @@
...
-Astro это **веб-фреймворк все-в-одном**, для разработки **быстрых, ориентированных на контент вебсайтов.**
+**Astro** - это веб-фреймворк для создания **веб-сайтов, ориентированных на контент**, таких как блоги, маркетинг и электронная коммерция. Astro наиболее известен своим новаторским [фронтенд-архитектурным подходом](/ru/concepts/islands/), направленным на снижение нагрузки и сложности JavaScript по сравнению с другими фреймворками. Если вам нужен сайт, который быстро загружается и имеет отличные показатели SEO, то Astro - это то, что вам нужно.
 
-Почему стоит выбрать Astro вместо другого веб-фреймворка?
-Вот пять основных принципов проектирования, которые объясняют, почему мы создали Astro,
-проблемы, для решения которых он существует, и почему Astro может быть лучшим выбором для вашего проекта или команды.
+## Особенности
 
-#### Astro ...
+*Astro - это универсальный веб-фреймворк.** Он включает в себя всё необходимое для создания веб-сайта. Также доступны сотни различных [интеграций](https://astro.build/integrations/) и [API-хуков](/ru/reference/integrations-reference/) для настройки проекта под ваши нужды и потребности.
```
Comment (baileys-li) [inline src/content/docs/ru/concepts/why-astro.mdx:11]: Maybe try [typograf](https://typograf.github.io/)
```suggestion
*Astro — это универсальный веб-фреймворк.** Он включает в себя всё необходимое для создания веб-сайта. Также доступны сотни различных [интеграций](https://astro.build/integrations/) и [API-хуков](/ru/reference/integrations-reference/) для настройки проекта под ваши нужды и потребности.
```

**Review by baileys-li — CHANGES_REQUESTED** [review]
I want to translate the documentation too and thanks for showing an example 👍 
I think my suggestions will make the text better for Russian-speaking readers

File: src/content/docs/ru/concepts/why-astro.mdx (hunk 1 above)
Comment (mearashadowfax) [reply src/content/docs/ru/concepts/why-astro.mdx:26]: Thanks for the review! If you check closer, the English version, or any other, uses 'Astro is. . .' not 'Astro...'. On the website it looks all the same.

**Review by baileys-li — APPROVED** [review]

**Review by yanthomasdev — APPROVED** [review]
LGTM! Great work peeps 🙌

**Pull request merged**